build-tools/packages/build-cli/src/handlers/promptFunctions.ts

```diff
@@ -262,6 +262,16 @@ export const promptToRelease: StateHandlerFunction = async (
     assert(context !== undefined, "Context is undefined.");
 
     const flag = isReleaseGroup(releaseGroup) ? "-g" : "-p";
+
+    const link =
+        releaseGroup === "client"
+            ? ADOPipelineLinks.CLIENT
+            : releaseGroup === "server"
+            ? ADOPipelineLinks.SERVER
+            : releaseGroup === "azure"
+            ? ADOPipelineLinks.AZURE
+            : ADOPipelineLinks.BUILDTOOLS;
```

Comment:
I suggest making this a function, because you need to include the pipelines for the individual packages like common-utils.